lib/kubeclient/common.rb

```diff
@@ -477,5 +482,16 @@ def http_options(uri)
 
       options.merge(@socket_options)
     end
+
+    def expired_token_message(http_code, message)
```

Comment:
@moolitayer intended to include the original message. That's why it passes in `message` and carries that through. Maybe I misunderstood your comment.

Another option would be to have a `TokenExpiredException` (or something) that takes the original exception as an argument and adds the expiration message. I thought this was a simpler approach.